File: Money.java
public class Money {
    private final int pesos;
    private final int cents;

    public Money(int pesos, int cents) {
        this.pesos = pesos;
        this.cents = cents;
    }

    public int pesos() {
        return this.pesos;
    }

    public int cents() {
        return this.cents;
    }

    @Override
    public String toString() {
        String zero = "";
        if (cents < 10) {
            zero = "0";
        }

        return "PHP " + pesos + "." + zero + cents;
    }

    public Money plus(Money addition) {
        int newPesos = this.pesos + addition.pesos;
        int newCents = this.cents + addition.cents;

        if (newCents >= 100) {
            newPesos += newCents / 100;
            newCents = newCents % 100;
        }

        return new Money(newPesos, newCents);
    }

    public boolean lessThan(Money compared) {
        int mergedMoney = (this.pesos * 100) + this.cents;
        int comparedMoney = (compared.pesos * 100) + compared.cents;

        return mergedMoney < comparedMoney;
    }

    public Money minus(Money decreaser) {
        int mergedMoney = (this.pesos * 100) + this.cents;
        int mergedCompared = (decreaser.pesos * 100) + decreaser.cents;

        int total = mergedMoney - mergedCompared;

        if (total < 0) {
            return new Money(0, 0);
        }

        int newPesos = total / 100;
        int newCents = total % 100;

        return new Money(newPesos, newCents);
    }
}

File: Practice/Money.cs
public class Money
{
    private int pesos;
    private int cents;

    public Money(int pesos, int cents)
    {
        this.pesos = pesos;
        this.cents = cents;
    }

    public int GetPesos()
    {
        return this.pesos;
    }

    public int GetCents()
    {
        return this.cents;
    }

    public int TotalCents()
    {
        return (this.pesos * 100) + this.cents;
    }

    public decimal GetTotal()
    {
        return (decimal) TotalCents() / 100;
    }

    public override string ToString()
    {
        String zero = "";
        if (cents < 10)
        {
            zero = "0";
        }
        return "PHP" + pesos + "." + zero + cents;
    }

    public void Received(Money addition)
    {
        int newPesos = this.pesos + addition.pesos;
        int newCents = this.cents + addition.cents;

        if (newCents >= 100)
        {
            newPesos += newCents / 100;
            newCents = newCents % 100;
        }

        this.pesos = newPesos;
        this.cents = newCents;
    }

    public void Paid(Money decreaser)
    {
        int mergedMoney = (this.pesos * 100) + this.cents;
        int mergedDecreaser = (decreaser.pesos * 100) + decreaser.cents;

        int total = mergedMoney - mergedDecreaser;

        if (total < 0)
        {
            this.pesos = 0;
            this.cents = 0;
        }

        int newPesos = total / 100;
        int newCents = total % 100;

        this.pesos = newPesos;
        this.cents = newCents;
    }

    public Money Difference(Money amount)
    {
        int totalDifference = this.TotalCents() - amount.TotalCents();
        int pesoDifference = totalDifference / 100;
        int centDifference = totalDifference % 100;

        return new Money(pesoDifference, centDifference);
    }
}